src/thing_objects.c

```diff
@@ -1750,7 +1749,110 @@ TngUpdateRet object_update_object_scale(struct Thing *objtng)
 
 TngUpdateRet object_update_power_sight(struct Thing *objtng)
 {
-    return _DK_object_update_power_sight(objtng);
+    int result; // eax
+
+    objtng->health = 2;
+
+    static const long effkind[] = {43, 86, 87, 88};
+    struct Dungeon * dungeon = get_dungeon(objtng->owner);
+
+    if ( !S3DEmitterIsPlayingSample(objtng->snd_emitter_id, 51, 0) ) {
+        thing_play_sample(objtng, 51, 0x64u, -1, 3u, 1u, 3, 256);
```

Comment:
3u and 1u could be normal numbers, NORMAL_PITCH is 0x64u, and FULL_LOUDNESS the last 256